### PR TITLE
Update on code and packages.json commandes

### PR DESCRIPTION
**Pull request by Mcastres**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,46 +11,69 @@ You may want to know how to develop such a starter by your own! This starter is
...
+# Using yarn
+yarn setup:yarn
 
+# Using npm
+npm run setup:npm
 ```
-cd backend
-strapi dev
+
+### Start the backend and frontend servers
```
Comment (soupette) [inline README.md]: I think that the `yarn develop` command should be used in order to quickly preview the expected result as when creating data in strapi the user will have to restart his client server so adding the seed in the command will be a great.
If you do so this heading should be:

`### Scaffold your project`

File: frontend/gatsby-config.js [hunk 2]
```diff
@@ -16,7 +20,7 @@ module.exports = {
     {
       resolve: "gatsby-source-strapi",
       options: {
-        apiURL: "http://localhost:1337",
+        apiURL: process.env.API_URL,
```
Comment (soupette) [inline frontend/gatsby-config.js]: Good it's not a good practice to commit the `.env` files and I haven't checked the update in the written tutorial but if the user forgets to create one you can add catch the error by doing the following:

```
process.env.API_URL || 'http://localhost:1337',
```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+    "setup:npm": "npm install && npm-run-all -p setup:npm:*",
+    "setup:npm:backend": "cd backend && npm install",
+    "setup:npm:frontend": "cd frontend && npm install",
+    "develop:npm": "npm-run-all -p develop:npm:*",
+    "develop:npm:backend": "cd backend && npm run develop",
+    "develop:npm:frontend": "cd frontend && npm run develop",
+    "setup:yarn": "yarn && npm-run-all -p setup:yarn:*",
+    "setup:yarn:backend": "cd backend && yarn",
+    "setup:yarn:frontend": "cd frontend && yarn",
+    "develop:yarn": "npm-run-all -p develop:yarn:*",
```
Comment (soupette) [inline package.json]: If you take the comment above in consideration you can add the seed command first:

```
"develop:yarn": "yarn seed:yarn && npm-run-all -p develop:yarn:*",
```

**Review by soupette — COMMENTED** [review]
According to your config you need to add the `src/images` folder in your gatsby app otherwise it throws an  error on boot.

File: README.md (hunk 1 above)
Comment (Mcastres) [reply README.md]: The seed is automatically executed by the CLI

File: package.json (hunk 3 above)
Comment (Mcastres) [reply package.json]: Same as upper comment, the seed is made by the CLI, not sure to run it every time you develop your app :/

**Pull request merged**